docs/getting_started.md

```diff
@@ -93,16 +93,13 @@ make install
 go run ./cmd/agent/main.go
 ```
 
-In the near future we will provide individual Docker images for each component,
-as well as the Kubernetes templates.
-
 ## Migrating from Zipkin
 
-Collector service exposes Zipkin compatible REST API `/api/v1/spans` and can be enabled by
-`--collector.zipkin.http-port=9411`. It supports Thrift and JSON format.
-Agent uses `TBinaryProtocol` and is available on `UDP` port `5775`.
+Collector service exposes Zipkin compatible REST API `/api/v1/spans` and `/api/v2/spans`.
+By default it's disabled. It can be enabled with `--collector.zipkin.http-port=9411`. 
+Agent uses `TBinaryProtocol` and it is available on `UDP` port `5775`.
```

Comment:
This is a bit terse, I would clarify that the agent can receive zipkin thrift over UDP. Having said that, not sure how useful it is as a migration mechanism, since zipkin clients typically don't use UDP. I think it would be great to achieve party between the agent and the collector for accepting spans over HTTP.

Reply:
I am not aware of any zipkin tracer/collector using UPD. Perhaps it's better to remove it from here because only jaeger clients are/were using it.